nimble/host/include/host/ble_hs.h

```diff
@@ -184,6 +184,22 @@ extern "C" {
 
 /** CSRK distibution and LinkKey are not supported */
 
+/**
+ * @}
+ */
+
+/**
+ * @brief LE deprecate warning
+ *
+ * @{
+ */
+
+#if MYNEWT_VAL(BLE_SUPPRESS_DEPRECATE_WARN)
```

Comment:
this header should include #include "syscfg/syscfg.h if it is using MYNEWT_VAL() makro

Reply:
Done